Components/alert/alert.css

```diff
@@ -0,0 +1,38 @@
+/* Alert CSS */
+
+.alert {
+    font-size: large;
+    width: 50%;
+    padding: 1rem;
+    color: white;
+    margin: 1rem;
+    background-color: #991B1B;
```

Comment:
It would be great if you use CSS variable here also, so your consistency won't break